README.md

```diff
@@ -17,6 +17,9 @@ HyperApp is a JavaScript library for building frontend applications.
 [Try it Online](https://codepen.io/hyperapp/pen/zNxZLP?editors=0010)
 
 ```jsx
+import { h, app } from 'hyperapp'
+/** @jsx h */
```

Comment:
This is transpiler specific. As the import statement LGTM to understand where `h` and `app` come from. But I dont think exposing pragma is a good thing.